avocado/plugins/list.py

```diff
@@ -165,6 +168,10 @@ def configure(self, parser):
                             'Current: %(default)s')
         loader.add_loader_options(parser)
 
+        filtering = parser.add_argument_group('filtering parameters')
+        filtering.add_argument('--filter-by-tags', metavar='TAGS',
+                               action='append')
```

Comment:
one more idea, we could use `nofoo,nofast` etc. But I still prefer the `-` if we document an example with `--filter-by-tags=-foo` somewhere visibly.